spark/src/main/java/org/apache/iceberg/spark/SparkReadOptions.java

```diff
@@ -48,6 +48,9 @@ private SparkReadOptions() {
   // Overrides the table's read.parquet.vectorization.batch-size
   public static final String VECTORIZATION_BATCH_SIZE = "batch-size";
 
+  // Overrides the table's read.locality.enabled
+  public static final String LOCALITY_ENABLED = "locality";
```

Comment:
What about `locality-enabled`?

Reply:
The locality was passed by spark read option ["locality"](https://github.com/apache/iceberg/blob/master/spark3/src/main/java/org/apache/iceberg/spark/Spark3Util.java#L497). Maybe someone is already using it?

Reply:
Okay, this was an existing setting? If so we don't need to rename it.